lib/package/config.coffee

```diff
@@ -77,12 +77,23 @@ config =
         type: 'string'
         default: ''
         order: 9
+      autoCompletionSuggestionPriority:
+        title: 'Auto-Completion Suggestion Priority'
+        description:
+          '''
+          Specify the sort order of Auto-completion suggestion from Julia-Client.
+          Note the default providers like snippets have priority of 1.
+          Requires Atom restart to take an effect.
+          '''
+        type: 'integer'
+        default: 3
+        order: 11
```

Comment:
Does this really need to be a setting?

Reply:
I personally uses snippets a lot and IMHO I want to have a control over priorities of completions like this.

But at the same time I doubt that most users have only 2 completion providers for Julia files (from Juno and snippet), and thus I this we can make this config into `boolean` one, which would appear simpler to user's view point.

Reply:
Fair. Let's keep it like this then.
I personally have snippets disabled all the time, but I guess giving the julia-client completions a higher priority would be even better.

Reply:
Cool. Maybe the default value is better to be more than `3`, but who knows.